server/reload.go

```diff
@@ -1063,6 +1063,23 @@ func (s *Server) diffOptions(newOpts *Options) ([]option, error) {
 				continue
 			}
 			fallthrough
+		case "noauthuser":
+			if oldValue != _EMPTY_ && newValue == _EMPTY_ {
+				for _, user := range newOpts.Users {
+					if user.Username == oldValue {
+						return nil, fmt.Errorf("config reload not supported for %s: old=%v, new=%v",
+							field.Name, oldValue, newValue)
+					}
+				}
+			} else {
+				return nil, fmt.Errorf("config reload not supported for %s: old=%v, new=%v",
+					field.Name, oldValue, newValue)
+			}
+		case "systemaccount":
+			if oldValue != DEFAULT_SYSTEM_ACCOUNT || newValue != _EMPTY_ {
```

Comment:
I assume we won't get here if oldValue == new Value.

Reply:
For this code no..